_posts/2021-01-13-hapi-fhir.md

```diff
@@ -175,44 +184,47 @@ import org.springframework.context.ApplicationContext;
 import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
-public class FhirexampleApplication {
+public class FhirExampleApplication{
 
     @Autowired
     private ApplicationContext context;
 
 
     public static void main(String[] args) {
 
-        SpringApplication.run(FhirexampleApplication.class, args);
+        SpringApplication.run(FhirExampleApplication.class, args);
     }
 
     @Bean
     public ServletRegistrationBean ServletRegistrationBean() {
-        ServletRegistrationBean registration= new ServletRegistrationBean(new FhirRestfullServer(context),"/*");
+        ServletRegistrationBean registration= new ServletRegistrationBean(new FhirRestfulServer(context),"/*");
         registration.setName("FhirServlet");
         return registration;
     }
 }
 ```
 
-Once your server has started, 
-Open up your Postman and enter this URL http://localhost:8080/metadata. 
-A capability statement will appear just like the screenshot below. 
-    Congratulations you are now FHIR enabled!
+Once your server has started, open up your Postman and GET the following URL: http://localhost:8080/metadata. 
+A capability statement will appear just like the screenshot below.
+A Capability Statement documents a set of capabilities (behaviors) of a FHIR Server for a particular version of FHIR that may be used as a statement of actual server functionality or a statement of required or desired server implementation. 
+Congratulations you are now FHIR enabled!
+
+<img class="image" style="margin:0px auto; max-width:80%" alt="postman" src="/img/2021-01-23-hapi-fhir/postman.png"/>
 
-<img class="image fit" alt="postman" src="/img/2021-01-23-hapi-fhir/postman.png">
 
 
 ### Resources
 
-The Patient provider is where the FHIR Patient behaviour is configured. 
-HAPI uses annotations to indicate what service the procedures provide. 
+The PatientProvider is where the FHIR Patient behaviour is configured. 
+HAPI uses annotations to indicate what kind of REST service a class method providesS. 
 The procedure createPatient in the diagram is annotated with @Create which indicates it handles POST/create. 
```

Comment:
1) add quotes for readability: The procedure "createPatient" in the diagram ...
2) ... which indicates it handles ... --> ... which indicates that it handles ...